### PR TITLE
CFY-5135 - support ssl on file server

### PR DESCRIPTION
**Pull request by noakup**

### CODE REVIEW

File: cloudify_cli/bootstrap/bootstrap.py [hunk 1]
```diff
@@ -314,6 +314,8 @@ def bootstrap(blueprint_path,
         manager_key_path = manager_node_instance.runtime_properties[
             MANAGER_KEY_PATH_RUNTIME_PROPERTY]
     else:
+        logger = get_logger()
+        logger.info('bootstrap outputs: {0}'.format(env.outputs()))
```
Comment (dankilman) [inline cloudify_cli/bootstrap/bootstrap.py]: remove

File: cloudify_cli/bootstrap/bootstrap.py [hunk 2]
```diff
@@ -563,7 +565,11 @@ def upload_plugin(plugin_path):
             _get_resource_into_dir(temp_dir, plugin_source_path,
                                    retries, wait_interval, timeout)
 
-        upload_plugin(plugin_local_path)
+        try:
```
Comment (dankilman) [inline cloudify_cli/bootstrap/bootstrap.py]: remove

File: cloudify_cli/utils.py [hunk 3]
```diff
@@ -411,6 +411,9 @@ def get_rest_host():
 get_management_server_ip = get_rest_host
 
 
+get_management_server_ip = get_rest_host
```
Comment (dankilman) [inline cloudify_cli/utils.py]: remove

File: cloudify_cli/commands/use.py [hunk 4]
```diff
@@ -31,6 +31,12 @@
 
 def use(management_ip, rest_port):
     logger = get_logger()
+
+    # check if cloudify was initialized.
+    if not utils.is_initialized():
```
Comment (dankilman) [inline cloudify_cli/commands/use.py]: cleanup jira

Comment (noakup) [inline cloudify_cli/commands/use.py]: [CFY-5752](https://cloudifysource.atlassian.net/browse/CFY-5752)

**Pull request merged**